gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/HttpDriverIntegrateTest.java

```diff
@@ -71,6 +69,21 @@ public Settings overrideSettings(final Settings settings) {
         return settings;
     }
 
+    @Test
```

Comment:
is this a real test? can it be removed? if not, better name for it?

Reply:
removed